src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachToCaseCallbackService.java

```diff
@@ -19,18 +19,17 @@
 import java.util.Map;
 
 import static io.vavr.control.Validation.valid;
+import static java.util.Arrays.asList;
 import static java.util.Collections.singletonList;
 import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.CaseReferenceTypes.CCD_CASE_REFERENCE;
 import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.SEARCH_CASE_REFERENCE_TYPE;
 import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.domains.envelopes.model.Classification.SUPPLEMENTARY_EVIDENCE_WITH_OCR;
 
 @Service
-public class AttachToCaseCallbackService {
+public class AttachToCaseCallbackService extends CallbackService {
```

Comment:
instead of extending we can create new until component?

Reply:
There are already 7 constructor parameters in CreateCaseCallbackService - max recommended number

Reply:
you are removing 2 of them

Reply:
I can not see too much benefit of touch these 2 classes.

Reply:
I agree. This is just code improvement. These 2 classes are doing similar things but in 2 different ways.
There is really not too much benefit, I will close this PR.